internal/helm/helm.go

```diff
@@ -239,7 +238,7 @@ func ResolveHelmChartVersion(versionFlag, chartDirectoryFlag, repository string)
 }
 
 func resolveChartVersion(versionFlag string, repository string) (semver2.Version, *chart.Chart, error) {
-	version, err := utils.ParseCiliumVersion(versionFlag)
+	version, err := semver2.ParseTolerant(versionFlag)
```

Comment:
nit: import alias to semver (to be consistent) or semver4

Reply:
oh yeah i took care of it in another pull request here. let me rebase 🚀🙏